templates/swift/Sources/Services/Service.swift.twig

```diff
@@ -22,9 +22,6 @@ open class {{ service.name | caseUcfirst | overrideIdentifier }}: Service {
     /// @throws Exception
     /// @return array
     ///
-    {%~ if method.type == "webAuth" %}
-    @available(iOS 14.0, macOS 11.0, tvOS 14.0, watchOS 7.0, *)
-    {%~ endif %}
```

Comment:
We'll need to duplicate this file into the apple templates and keep this in the duplicate